### ISSUE TITLE
CSP error

### ISSUE THREAD

**Issue by an-workramp** [hellosign/hellosign-embedded#151]
I am using the latest version of `embedded.production.min` as follows:

```
<!DOCTYPE html>
<html>
<head>
<script src="/embedded.production.min"></script>
</head>
<body>

<script>
client = new window.HelloSign({
  clientId: '93a06de3c2a68f21bf92e609cd6bee20'
});

client.open('https://app.hellosign.com/editor/embeddedTemplate?token=3215bdc75e94ebc53bb5594ce15e9bac&amp;root_snapshot_guids%5B0%5D=482054d2c26f2cfe08168b9c2dfee9a4af48f80b&amp;snapshot_access_guids%5B0%5D=f51a2bd3&amp;guid=31478bd3d5bf5f3c2e85ba4805b67de3e45ce9f6', {
  testMode: true
});
</script>

</body>
</html>
```

A frame pops up with the message "Missing parameter, no ID" and I get the following errors in console:

<img width="869" alt="Screen Shot 2020-04-08 at 2 25 49 PM" src="https://user-images.githubusercontent.com/55512956/78830288-ed3cbe80-79a4-11ea-9596-11685f6b48d9.png">

The page's CSP header value is set to `script-src 'self' https: 'unsafe-inline' 'unsafe-eval' cdn.hellosign.com; frame-src 'self' https: 'unsafe-inline' 'unsafe-eval' cdn.hellosign.com; default-src 'self' https: 'unsafe-inline' 'unsafe-eval'`

Then, when I click the green "Continue" at the top of the editor, I get the error `Could not save! DOMException: "Permission denied to access property "protocol" on cross-origin object"`.

<img width="1138" alt="Screen Shot 2020-04-09 at 8 57 21 AM" src="https://user-images.githubusercontent.com/55512956/78909068-3130d180-7a40-11ea-99b7-95c6a8966e4e.png">

**Comment by an-workramp** [comment]
To add to this, when I click the green "Continue" at the top of the editor, I get the error `Could not save! DOMException: "Permission denied to access property "protocol" on cross-origin object"`.

<img width="1138" alt="Screen Shot 2020-04-09 at 8 57 21 AM" src="https://user-images.githubusercontent.com/55512956/78909068-3130d180-7a40-11ea-99b7-95c6a8966e4e.png">

**Comment by janetanne** [comment]
this was resolved via ticket #10512043.

thanks for working with us @an-workramp !
